### DISCUSSION TITLE
Different UI output in running command "npm run dev" and "npm run start" after "npm run build"

### DISCUSSION THREAD

**Discussion by ProPhycient5** [tailwindlabs/tailwindcss#5029] [Help] [answered]
@ALL Can anyone plz tell, How can I fix the problem in getting some different UI output while running "npm run start" with no error, but I am getting usual UI output while running "npm run dev", after "npm run build" ??????

I am doing Next-ReactJS project with tailwindCSS .

Although I think I have to fix this error in postcss.config.js file, but I don't know what changes particularly, I should make.
Please help me 🙏🏼

this is my postcss.config.js file.
```
module.exports = {
  plugins: {
    'postcss-import': {},
    tailwindcss: {},
    ...(process.env.NODE_ENV === 'production'
      ? {
        '@fullhuman/postcss-purgecss': {
          content: ['./pages/**/*.{js,jsx,ts,tsx}', './components/**/*.{js,jsx,ts,tsx}'],
          defaultExtractor: content => content.match(/[A-Za-z0-9-_:/]+/g) || [],
        },
      }
      : {}),
    'postcss-preset-env': { stage: 2 },
  },
};
```

**Comment by simonswiss** [comment, marked as answer]
Hey!

If you're using a relatively recent version of Tailwind CSS (since version 1.4 I believe), you don't need so specify Purge CSS in your Post CSS config file. It's all available in core via the Tailwind config file:

https://tailwindcss.com/docs/optimizing-for-production#basic-usage

Are you able to share the code for your project in a GitHub repo?

**Reply by ProPhycient5** [reply]
Thank you, for your reply. I was able to resolve by keeping default code in postcss.config.js file.

**Reply by simonswiss** [reply]
Glad I was able to help 👍
